README.md

```diff
@@ -14,6 +14,7 @@ The goal of this project is to simplify the process of calculating and filing th
 
 Currently, the following brokers are supported:
 - Interactive Brokers (last checked: 28 March 2022) [(guide)](docs/brokers/interactive-brokers-guide.md)
+- Degiro (last checked: 2 April 2023) [(guide)](docs/brokers/degiro-guide.md)
```

Comment:
Could you change this to "DEGIRO" here as well in the guide itself to stay consistent with the brand itself?

Reply:
Resolved display of DEGIRO brand name in documentation.
What wasn't reviewed thought:
- Name in the netlify URL (all lowercase, consistent with `/guide-ibkr`)
- Name in the docs folder (all lowercase, consistent with `interactive-brokers-guide`)